### PR TITLE
setup linter

### PR DESCRIPTION
**Pull request by devinburnette**
sets up betterlint rubocop as a linter

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+inherit_gem:
+  betterlint:
+    - config/default.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
+AllCops:
+  TargetRubyVersion: 2.6
```
Comment (smudge) [inline .rubocop.yml]: We could probably do something more recent, but that would require updating `spec.required_ruby_version` in the gemspec as well.

File: rspec-fortify.gemspec [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+  spec.license       = 'MIT'
+
+  spec.files         = `git ls-files`.split($OUTPUT_RECORD_SEPARATOR)
+  spec.executables   = []
+  spec.name          = 'rspec-fortify'
+  spec.require_paths = ['lib']
+  spec.version       = RSpec::Fortify::VERSION
+  spec.required_ruby_version = '>= 2.6'
+  spec.add_runtime_dependency 'rspec-core', '>3.9'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```
Comment (smudge) [inline rspec-fortify.gemspec]:
```suggestion
  spec.license       = 'MIT'
  spec.metadata['rubygems_mfa_required'] = 'true'


  spec.files         = `git ls-files`.split($OUTPUT_RECORD_SEPARATOR)
  spec.executables   = []
  spec.name          = 'rspec-fortify'
  spec.require_paths = ['lib']
  spec.version       = RSpec::Fortify::VERSION
  spec.required_ruby_version = '>= 2.6'
  spec.add_runtime_dependency 'rspec-core', '>3.9'
```

Nitpick, but the autocorrect always adds it to the bottom, and it should probably go at the bottom of the top half.

File: Rakefile [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 begin
   require 'rspec/core/rake_task'
 
   RSpec::Core::RakeTask.new(:spec)
 
-  task :default => :spec
-rescue LoadError
+  task default: :spec
+rescue LoadError # rubocop:disable Lint/SuppressedException
 end
```
Comment (smudge) [inline Rakefile]: This begin...rescue block shouldn't be necessary. Possibly a vestige of a much older way of loading gem dependencies.

File: lib/rspec/fortify.rb [hunk 4]
```diff
@@ -229,7 +229,7 @@ def should_retry?
 end
 
 def cast_to_boolean(value)
-  if value.nil? || %w(false f 0 no n).include?(value.to_s.downcase)
+  if value.nil? || %w(false f 0 no n).include?(value.to_s.downcase) # rubocop:disable Style/IfWithBooleanLiteralBranches
```
Comment (smudge) [inline lib/rspec/fortify.rb:232]: Oh, interesting bug in the linter I think.

File: .rubocop.yml (hunk 1 above)
Comment (devinburnette) [reply .rubocop.yml]: how recent you think we should go?

Comment (smudge) [reply .rubocop.yml]: 3.0 is probably fine.

Comment (smudge) [reply .rubocop.yml]: That will likely produce some additional linter flags and autocorrects.

**Review by smudge — APPROVED** [review]
domain LGTM && platform LGTM -- I assume the GH Action will come in a follow-up PR?

**Pull request merged**